integration-tests/tests/ubuntuFan_test.go

```diff
@@ -1,5 +1,5 @@
 // -*- Mode: Go; indent-tabs-mode: t -*-
-// +build !excludeintegration
```

Comment:
@zyga good catch thanks, I've added a note on integration-tests/README.md about the excludeflaky tag, currently is the only one passed to the integration tests.

excludeintegration is used at a higher level for having separated unit and integration tests, not sure if/where we should document this.